src/fixtures/comment.ts

```diff
@@ -50,7 +41,7 @@ export const comment = {
           "https://discussion.guardianapis.com/discussion-api/comment/138809272",
         isoDateTime: "2020-03-09T19:09:28Z",
         date: "09 March 2020 7:09pm",
-        commentId: "138809272",
+        commentId: 138809272,
```

Comment:
Is this part of the move or the start of something else?

Reply:
for the sake of keeping this PR small im moving everything to number, in the next PR I will move them to all strings (right not they error types)